dyne/examples.py

```diff
@@ -533,3 +534,158 @@ def h(k, X, with_jacobian=True):
 
     return NonlinearProblemExample(X0, P0, f, Q, [(measurement_epochs, Z, h, R)],
                                    n_epochs, Xt, Wt)
+
+
+def generate_magnetic_heading(
+    total_time=120,
+    time_step=0.1,
+    rph_mean=[3, -5, 0],
+    rph_change_amplitude=5,
+    rph_change_period=[5, 5, 30],
+    rph_change_phase_offset=[0, 90, 45],
+    magnetic_field_n=[10, 3, 40],
+    mag_bias=[30, -20, 70],
+    X0=None,
+    P0=np.diag([180, 50, 50, 50])**2,
+    q=np.zeros(4),
+    measurement_sd=[1, 1, 1],
+    rng=0
+):
+    """Generate data for exapmle of heading estimation using biased magnetometer data.
+
+    Roll, pitch and heading angles are varying according to a given harmonic law.
+    Exact values of roll and pitch are known, heading and magnetometer biases
+    should be estimated.
```

Comment:
I think a better phrasing is "are to be estimated".

Reply:
done